### ISSUE TITLE
'forms' is an undeclared prefix

### ISSUE THREAD

**Issue by einc** [martijn00/XamarinMediaManager#218]
Hello,
I am trying to implement the MediaForms sample (https://github.com/martijn00/XamarinMediaManager/blob/develop/Samples/Forms/MediaForms/MediaFormsPage.xaml)
However, I get the error:
'forms' is an undeclared prefix. Line 16, position 14.
Here is the code for it:

         <forms:VideoView Grid.Row="0" Grid.RowSpan="1" HorizontalOptions="FillAndExpand" VerticalOptions="FillAndExpand"

                     Source="http://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4" AspectMode="AspectFill"/>
I am using Xamarin Forms for Visual Studios 2017. I have downloaded Plugin.MediaManager and Plugin.MediaManager.Forms version 0.4.0 for both my Portable class and Android class.

In my App.xaml.cs, I have added:
        public App()
        {
            InitializeComponent();
            var workaround = typeof(VideoView);
            MainPage = new MediaFormsPage();
        }

In my Main Activity, I have added:
VideoViewRenderer.Init();

and the necessary references.
I'm not sure why it is not working

**Comment by tankitty** [comment]
add this namespance

  xmlns:forms="clr-namespace:Plugin.MediaManager.Forms;assembly=Plugin.MediaManager.Forms"

**Comment by einc** [comment]
I added the namespace and now I can build (there are no errors)  and see the buttons. However, the video does not load or is not there and when I press the play button nothing happens.

This is the code I have for the play button: 
void PlayClicked(object sender, System.EventArgs e)
        {
            PlaybackController.Play();
        }

I am using an Android 4.0.3 API 15 to test. I turned on Write External Storage and Read External Storage in Android Manifest; is there another required permission I need?

what I currently see on my phone:
![emulator](https://user-images.githubusercontent.com/14082828/28476750-ff1fd61c-6e05-11e7-9150-9de880a6607e.jpg)
